doc/source/installing.rst

```diff
@@ -38,22 +38,22 @@ To install using pip, simply run:
 You may also need to install some system dependencies (depending on your host).
 These are:
 
-- `OpenSSL <https://openssl.org>`_ used to generate and verify RSA signatures, and to export and verify signatures created with gpg.
-- `GPG <https://gnupg.org>`_ if you plan on generating gpg signatures (verifying
-  works without gpg).
+- `OpenSSL <https://openssl.org>`_ used to generate and verify RSA signatures, and to export and verify signatures created with GPG.
+- `GPG <https://gnupg.org>`_ if you plan on generating GPG signatures (verifying
+  works without GPG).
 
 Installing from Source
 ----------------------
 
-If your system doesn't provide in-toto yet, you can also install it from the
+If your system doesn't yet provide in-toto, you can also install it from the
```

Comment:
I'm not completely convinced moving this yet is necessary? I'm also wondering if we could simplify this:

```
if your system doesn't provide in-toto, you can install it from the source
```

to be a little bit more succint.